src/myr/modelReference.js

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import {
+    Tooltip,
+    Typography,
+    withStyles
+} from "@material-ui/core";
+
+const HtmlTooltip = withStyles(theme => ({
+    tooltip: {
+        maxWidth: 300,
+        fontSize: theme.typography.pxToRem(14),
+        "& b": {
+            fontWeight: "inherit",
+        },
+    },
+}))(Tooltip);
+
+export const stringText = (text) => {
```

Comment:
We'll want to eventually move `<type>Text` to it's own file to prevent redundant definitions. So reference fields would import this file in to use the type texts.